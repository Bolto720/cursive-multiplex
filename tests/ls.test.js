const { setup } = require('shellshot');

setup();

it(
    'checks if the files in this directory are the same',
    async () => {
        await expect.command('ls')
            .forStdout(expectation => expectation.toMatchSnapshot());
    },
);
