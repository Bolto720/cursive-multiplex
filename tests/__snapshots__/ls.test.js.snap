// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if the files in this directory are the same 1`] = `
"__snapshots__
end2end.rs
ls.test.js
node_modules
package.json
tree.rs
"
`;
